
DEBUG: Running node-supervisor with
DEBUG:   program 'gomemfill.js'
DEBUG:   --watch 'memfill.js,gomemfill.js'
DEBUG:   --extensions 'node|js'
DEBUG:   --exec 'node'

DEBUG: Starting child process with 'node gomemfill.js'
DEBUG: Watching directory '/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js' for changes.
DEBUG: Watching directory '/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/gomemfill.js' for changes.
memfill Hello

memfill Error: console.error invocation
    at Error (unknown source)
    at Object.myConsoleError (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:37:11)
    at Object.<anonymous> (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:9:9)
    at Module._compile (module.js:441:26)
    at Object..js (module.js:459:10)
    at Module.load (module.js:348:31)
    at Function._load (module.js:308:12)
    at Array.0 (module.js:479:10)
    at EventEmitter._tickCallback (node.js:192:40)

node-http nodeapi-http

node-http listening at: localhost:1080

node-http 

node-http node.js:201

node-http         throw e; // process.nextTick error, or 'error' event on first tick

node-http               
node-http ^

node-http Error: listen EADDRINUSE
    at errnoException (net.js:670:11)
    at Array.0 (net.js:771:26)
    at EventEmitter._tickCallback (node.js:192:40)

node-http exit: 1
memfill FATAL ERROR: JS Allocation failed - process out of memory

memfill exit: 1
DEBUG: Program node gomemfill.js exited with code 0

DEBUG: Starting child process with 'node gomemfill.js'
memfill Hello

memfill Error: console.error invocation
    at Error (unknown source)
    at Object.myConsoleError (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:37:11)
    at Object.<anonymous> (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:9:9)
    at Module._compile (module.js:441:26)
    at Object..js (module.js:459:10)
    at Module.load (module.js:348:31)
    at Function._load (module.js:308:12)
    at Array.0 (module.js:479:10)
    at EventEmitter._tickCallback (node.js:192:40)

node-http nodeapi-http

node-http listening at: localhost:1080

node-http 

node-http node.js:201

node-http         throw e; // process.nextTick error, or 'error' event on first tick

node-http  
node-http              ^

node-http Error: listen EADDRINUSE
    at errnoException (net.js:670:11)
    at Array.0 (net.js:771:26)
    at EventEmitter._tickCallback (node.js:192:40)

node-http exit: 1
memfill FATAL ERROR: JS Allocation failed - process out of memory

memfill exit: 1
DEBUG: Program node gomemfill.js exited with code 0

DEBUG: Starting child process with 'node gomemfill.js'
memfill Hello

memfill Error: console.error invocation
    at Error (unknown source)
    at Object.myConsoleError (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:37:11)
    at Object.<anonymous> (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:9:9)
    at Module._compile (module.js:441:26)
    at Object..js (module.js:459:10)
    at Module.load (module.js:348:31)
    at Function._load (module.js:308:12)
    at Array.0 (module.js:479:10)
    at EventEmitter._tickCallback (node.js:192:40)

node-http nodeapi-http

node-http listening at: localhost:1080

node-http 

node-http node.js:201

node-http         throw e; // process.nextTick error, or 'error' event on first tick

node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http ^
node-http 

node-http Error: listen EADDRINUSE
    at errnoException (net.js:670:11)
    at Array.0 (net.js:771:26)
    at EventEmitter._tickCallback (node.js:192:40)

node-http exit: 1
memfill FATAL ERROR: JS Allocation failed - process out of memory

memfill exit: 1
DEBUG: Program node gomemfill.js exited with code 0

DEBUG: Starting child process with 'node gomemfill.js'
memfill Hello

memfill Error: console.error invocation
    at Error (unknown source)
    at Object.myConsoleError (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:37:11)
    at Object.<anonymous> (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:9:9)
    at Module._compile (module.js:441:26)
    at Object..js (module.js:459:10)
    at Module.load (module.js:348:31)
    at Function._load (module.js:308:12)
    at Array.0 (module.js:479:10)
    at EventEmitter._tickCallback (node.js:192:40)

node-http nodeapi-http

node-http listening at: localhost:1080

node-http 

node-http node.js:201

node-http         throw e; // process.nextTick error, or 'error' event on first tick

node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http ^
node-http 

node-http Error: listen EADDRINUSE
    at errnoException (net.js:670:11)
    at Array.0 (net.js:771:26)
    at EventEmitter._tickCallback (node.js:192:40)

node-http exit: 1
memfill FATAL ERROR: JS Allocation failed - process out of memory

memfill exit: 1
DEBUG: Program node gomemfill.js exited with code 0

DEBUG: Starting child process with 'node gomemfill.js'
memfill Hello

memfill Error: console.error invocation
    at Error (unknown source)
    at Object.myConsoleError (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:37:11)
    at Object.<anonymous> (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:9:9)
    at Module._compile (module.js:441:26)
    at Object..js (module.js:459:10)
    at Module.load (module.js:348:31)
    at Function._load (module.js:308:12)
    at Array.0 (module.js:479:10)
    at EventEmitter._tickCallback (node.js:192:40)

node-http nodeapi-http

node-http listening at: localhost:1080

node-http 

node-http node.js:201

node-http         throw e; // process.nextTick error, or 'error' event on first tick

node-http               ^

node-http Error: listen EADDRINUSE
    at errnoException (net.js:670:11)
    at Array.0 (net.js:771:26)
    at EventEmitter._tickCallback (node.js:192:40)

node-http exit: 1
memfill FATAL ERROR: JS Allocation failed - process out of memory

memfill exit: 1
DEBUG: Program node gomemfill.js exited with code 0

DEBUG: Starting child process with 'node gomemfill.js'
memfill Hello

memfill Error: console.error invocation
    at Error (unknown source)
    at Object.myConsoleError (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:37:11)
    at Object.<anonymous> (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:9:9)
    at Module._compile (module.js:441:26)
    at Object..js (module.js:459:10)
    at Module.load (module.js:348:31)
    at Function._load (module.js:308:12)
    at Array.0 (module.js:479:10)
    at EventEmitter._tickCallback (node.js:192:40)

node-http nodeapi-http

node-http listening at: localhost:1080

node-http 

node-http node.js:201

node-http         throw e; // process.nextTick error, or 'error' event on first tick

node-http               
node-http ^

node-http Error: listen EADDRINUSE
    at errnoException (net.js:670:11)
    at Array.0 (net.js:771:26)
    at EventEmitter._tickCallback (node.js:192:40)

node-http exit: 1
memfill FATAL ERROR: JS Allocation failed - process out of memory

memfill exit: 1
DEBUG: Program node gomemfill.js exited with code 0

DEBUG: Starting child process with 'node gomemfill.js'
memfill Hello

memfill Error: console.error invocation
    at Error (unknown source)
    at Object.myConsoleError (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:37:11)
    at Object.<anonymous> (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:9:9)
    at Module._compile (module.js:441:26)
    at Object..js (module.js:459:10)
    at Module.load (module.js:348:31)
    at Function._load (module.js:308:12)
    at Array.0 (module.js:479:10)
    at EventEmitter._tickCallback (node.js:192:40)

node-http nodeapi-http

node-http listening at: localhost:1080

node-http 

node-http node.js:201
        throw e; // process.nextTick error, or 'error' event on first tick
              ^

node-http Error: listen EADDRINUSE
    at errnoException (net.js:670:11)
    at Array.0 (net.js:771:26)
    at EventEmitter._tickCallback (node.js:192:40)

node-http exit: 1
memfill FATAL ERROR: JS Allocation failed - process out of memory

memfill exit: 1
DEBUG: Program node gomemfill.js exited with code 0

DEBUG: Starting child process with 'node gomemfill.js'
memfill Hello

memfill Error: console.error invocation
    at Error (unknown source)
    at Object.myConsoleError (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:37:11)
    at Object.<anonymous> (/home/foxyboy/Desktop/c505/node/nodejs3/snippets/modules/supervisor/memfill.js:9:9)
    at Module._compile (module.js:441:26)
    at Object..js (module.js:459:10)
    at Module.load (module.js:348:31)
    at Function._load (module.js:308:12)
    at Array.0 (module.js:479:10)
    at EventEmitter._tickCallback (node.js:192:40)

node-http nodeapi-http

node-http listening at: localhost:1080

node-http 

node-http node.js:201

node-http         throw e; // process.nextTick error, or 'error' event on first tick

node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http  
node-http ^
node-http 

node-http Error: listen EADDRINUSE
    at errnoException (net.js:670:11)
    at Array.0 (net.js:771:26)
    at EventEmitter._tickCallback (node.js:192:40)

node-http exit: 1
